src/module/orbbec.cpp

```diff
@@ -1116,18 +1117,51 @@ vsdk::Camera::properties Orbbec::get_properties() {
                 buffer << service_name << ": device with serial number " << serial_number << " is not started";
                 throw std::invalid_argument(buffer.str());
             }
-            props = my_dev->pipe->getCameraParam().rgbIntrinsic;
+            auto const camera_params = my_dev->pipe->getCameraParam();
+            intrinsic_props = camera_params.rgbIntrinsic;
+            distortion_props = camera_params.rgbDistortion;
         }
 
         vsdk::Camera::properties p{};
         p.supports_pcd = true;
-        p.intrinsic_parameters.width_px = props.width;
-        p.intrinsic_parameters.height_px = props.height;
-        p.intrinsic_parameters.focal_x_px = props.fx;
-        p.intrinsic_parameters.focal_y_px = props.fy;
-        p.intrinsic_parameters.center_x_px = props.cx;
-        p.intrinsic_parameters.center_y_px = props.cy;
-        // TODO: Set distortion parameters
+        p.intrinsic_parameters.width_px = intrinsic_props.width;
+        p.intrinsic_parameters.height_px = intrinsic_props.height;
+        p.intrinsic_parameters.focal_x_px = intrinsic_props.fx;
+        p.intrinsic_parameters.focal_y_px = intrinsic_props.fy;
+        p.intrinsic_parameters.center_x_px = intrinsic_props.cx;
+        p.intrinsic_parameters.center_y_px = intrinsic_props.cy;
+
+        switch (distortion_props.model) {
+            case OB_DISTORTION_NONE:
+                p.distortion_parameters.model = "NONE";
+                break;
+            case OB_DISTORTION_MODIFIED_BROWN_CONRADY:
+                p.distortion_parameters.model = "MODIFIED_BROWN_CONRADY";
+                break;
+            case OB_DISTORTION_INVERSE_BROWN_CONRADY:
+                p.distortion_parameters.model = "INVERSE_BROWN_CONRADY";
+                break;
+            case OB_DISTORTION_BROWN_CONRADY:
+                p.distortion_parameters.model = "BROWN_CONRADY";
+                break;
+            case OB_DISTORTION_BROWN_CONRADY_K6:
+                p.distortion_parameters.model = "BROWN_CONRADY_K6";
+                break;
+            case OB_DISTORTION_KANNALA_BRANDT4:
+                p.distortion_parameters.model = "KANNALA_BRANDT4";
```

Comment:
Q tangential to the goal of this PR: where are we getting these strings from? could we import them as enums instead?